lua/entities/gmod_wire_expression2/core/serverinfo.lua

```diff
@@ -60,19 +60,19 @@ e2function number airDensity()
 end
 
 e2function number maxFrictionMass()
-	return physenv.GetPerformanceSettings()["MaxFrictionMass"]
+	return physenv.GetPerformanceSettings().MaxFrictionMass or 0
```

Comment:
I agree with making sure that all functions do return the type they say they will, but I don't agree that we should be assuming builtin functions might return garbage. Adding the `or 0` here gives the impression that `physenv.GetPerformanceSettings().MaxFrictionMass` might not be a number, which is misleading.